### DISCUSSION TITLE
Button to copy invitation link to clipboard does not work

### DISCUSSION THREAD

**Discussion by muchcodesuchwow** [sysadminsmedia/homebox#343] [Support] [answered]
### First Check

- [x] This is not a feature request
- [x] I added a very descriptive title to this issue.
- [x] I used the GitHub search to find a similar issue and didn't find it.
- [x] I searched the documentation, with the integrated search.
- [x] I already read the docs and didn't find an answer.
- [x] I can replicate the issue inside the Demo install.

### Homebox Version

0.15.2 - Build: c6158e7c9ebf8ce4c921078b71f71fe2e16606c4

### What is the issue you are experiencing?

The copy to clipboard button for the invitation link and the token alone does not work in my installation. But it works inside the demo server https://demo.homebox.software/profile (i can not replicate the issue inside the Demo install, however i have to check the checkbox to submit the issue.)
This looks mysterious to me.

Is this somehow linked to #211 ?

Tested with edge and firefox on Windows.

### How can the maintainer reproduce the issue?

Try to copy the inviation link to the clipboard using the button found unter /profile.
It does work in the Demo, but not in my installation.

### Deployment

Docker (Linux)

### OS Architechture

x86_64 (AMD, Intel)

**Comment by hay-kot** [comment]
The copy API in the browser is only accessible on localhost or over https. So if you are accessing a server by IP and not using https the copy buttons won’t work. 

This is likely the problem in the issue you linked as well.

**Reply by muchcodesuchwow** [reply]
Yes, i can confirm this. After putting it behind a reverse proxy it works.

**Comment by tankerkiller125** [comment, marked as answer]
I'm going to go ahead and close this.

For anyone who stumbles across this in the future (with the same issue):

After some thorough testing, @hay-kot is correct, the copy APIs are only available in HTTPs, when using HTTPs for the site copy works perfectly. When using localhost with HTTP though it fails. This is the expected behavior do to browser API implementations.

**Reply by tonyaellie** [reply]
Might be worth mentioning this in the docs and either hiding the button or showing a popup when the button is clicked when on http?
